### PR TITLE
Make lighthouse url configurable in example app

### PR DESCRIPTION
**Pull request by andrewthauer**
## Hey, I just made a Pull Request!

This updates the example app so the lighthouse url is configurable from the `app-config.yaml`

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: packages/app/src/apis.ts [hunk 1]
```diff
@@ -114,7 +114,10 @@ export const apis = (config: ConfigApi) => {
 
   builder.add(featureFlagsApiRef, new FeatureFlags());
 
-  builder.add(lighthouseApiRef, new LighthouseRestApi('http://localhost:3003'));
+  builder.add(
+    lighthouseApiRef,
+    new LighthouseRestApi(config.getString('lighthouse.baseUrl')),
```
Comment (Rugvip) [inline packages/app/src/apis.ts:119]: I'd vote for a `LighthouseRestApi.fromConfig` variant instead. Not too excited about the idea of these config paths diverging for different apps. Better code it into the plugin itself imo.

**Review by Rugvip — COMMENTED** [review]
Thanks!

File: packages/app/src/apis.ts (hunk 1 above)
Comment (andrewthauer) [reply packages/app/src/apis.ts]: @Rugvip - Let me know if this was what you had in mind.

**Review by freben — APPROVED** [review]
Looks good!

File: packages/app/src/apis.ts (hunk 1 above)
Comment (Rugvip) [reply packages/app/src/apis.ts]: @andrewthauer, yep, exactly that, thanks! :grin:

**Pull request merged**